test/concrete.jl

```diff
@@ -1,4 +1,7 @@
 @testset "import" begin
+    @test pymoduleexists("sys")
+    @test pymoduleexists("os")
+    @test !pymoduleexists(randstring(32))
```

Comment:
```suggestion
    @test !pymoduleexists("rDJb5uy3eTorFw4c7UbcGOC59FjVGcnB") # randstring(32)
```
No reason to choose a different random string for each test run.

Reply:
(Then you can also remove the `Random` dependency.)